nemo_skills/prompt/config/judge/code.yaml

```diff
@@ -0,0 +1,82 @@
+system: ""
+
+# Defines the judge prompt for code generation tasks
+# You need to use keys you want to use from the input dict using {}
+# For example, you generally want {instruction} and {output} keys for genetic instruct
+# Allowed keys for judgement - {Yes or No}, {Yes}, {No}. Other keys must be explicitly escaped in code.
+user: |
+  '''
+  You are an expert python programmer. 
+
+  Below is a question and code solution. Decide if the solution follows the below criteria and give a final Yes/No, and place it in the <judge></judge> tags.
+  
+  Only look at the function generated, not any examples/print statements etc. Just the core logic.
+  
+  Please first briefly describe your reasoning (in less than 30 words) without writing any code, and then write Decision: \\\\boxed{{Yes or No}} in your last line.
```

Comment:
are you sure you need to have \\\\ here? This yaml string shouldn't need any escaping, so this will render as actual 4 backslashes, not two if that's what you intended. And presumably, this should just be a single backslash like in the math config https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/prompt/config/generic/math.yaml#L15

Reply:
Yeah our judge uses 4 \.  I suppose we can change the code to make it do just 1. Don't remember why we chose \. Probably Mixtral 8x22b wasn't working well so we used this in few shot and if just stuck. Id leave it for now